guides/17_Migration_to_3_0.md

```diff
@@ -16,68 +16,156 @@ npm install amazon-chime-sdk-js@beta
 
 __Version 3 of the Amazon Chime SDK for JavaScript makes a number of interface changes.__
 
-__In many cases you should not need to adjust your application code at all. This will be the case if:__
+## Device controller
```

Comment:
Do we need to add the guide for device permission change and video synthesizing change?

Reply:
The migration guide has a section for moving the synthesize API out from the SDK.
https://github.com/aws/amazon-chime-sdk-js/blob/caae4102c90d8221014c0c404c2b635b64996309/guides/17_Migration_to_3_0.md#removing-synthesize-video-api

Reply:
Oops, I missed that. Thanks